tools/internal_ci/macos/grpc_objc_bazel_test.sh

```diff
@@ -119,3 +119,33 @@ objc_bazel_tests/bazel_wrapper \
   -- \
   "${EXAMPLE_TARGETS[@]}" \
   "${TEST_TARGETS[@]}"
+
+
+# Enable event engine and run tests again.
+TEST_TARGETS=(
```

Comment:
nit: let's call this EVENT_ENGINE_TEST_TARGETS  instead and update the command below instead of redefining TEST_TARGETS?